datajoint/schema.py

```diff
@@ -223,8 +229,4 @@ def external_table(self):
         """
         if self._external is None:
             self._external = ExternalTable(self.connection, self.database)
-        return self._external
-
-    def erd(self):
```

Comment:
Why is this getting removed? Given it was already part of the interface, would need a good reason for a removal, and I don't think this was discussed.

Reply:
I don't think `schema.erd` was used much.  It was never documented and it works differently from all other ways to produce the ERD.

Reply:
Ok, as many things in DataJoint has been **undocumented**, I usually don't like removing a feature based on that argument. There has been a lot of things that were *unofficially documented*, say by word of mouth etc. That being said, we do provide clear documentation on how to use the alternative method of `dj.ERD`. There may be still some utility in schema and even class level `erd` methods, but for consistency, I think I can be fine with the removal.